projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -131,24 +131,18 @@ export class DocumentDetailViewComponent implements DetailRecord, OnInit {
     if (undefined === provisions) {
       return [];
     }
+
     const results = {};
-    provisions.forEach((element: any) => {
+    provisions.map((element: any) => {
       const type =  element.type;
-      if (!(type in results)) {
-        results[type] = [];
-      }
-      if ('_text' in element) {
+      if (('_text' in element)) {
```

Comment:
is it necessary to have double parentheses?

Reply:
done